<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="result_screen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #d3d3d3;  
    border-radius: 25px; 
    width: 50px; 
    height: 50px;  
	 color: black; 
    font-size: 18px; 
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert_op">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #d3d3d3; 
    border-radius: 25px;  
    width: 50px; 
    height: 50px; 
	 color: black;  
    font-size: 18px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>+ / -</string>
   </property>
  </widget>
  <widget class="QPushButton" name="percentage_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #d3d3d3;  
    border-radius: 25px; 
    width: 50px; 
    height: 50px;  
	 color: black;  
    font-size: 18px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="divide_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFD700;  
    border-radius: 25px;  
    width: 50px; 
    height: 50px;  
	 color: white;  
    font-size: 27px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>รท</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiply_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFD700;  
    border-radius: 25px; 
    width: 50px; 
    height: 50px;  
	 color: white;  
    font-size: 23px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px;  
    width: 50px; 
    height: 50px;  
	 color: white;  
    font-size: 24px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9; 
    border-radius: 25px; 
    width: 50px;  
    height: 50px; 
	 color: white; 
    font-size: 24px; 
    font-weight: bold; 
}

</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9; 
    border-radius: 25px; 
    width: 50px; 
    height: 50px;  
	 color: white;  
    font-size: 24px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFD700; 
    border-radius: 25px;  
    width: 50px;  
    height: 50px;  
	 color: white; 
    font-size: 27px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px; 
    width: 50px;  
    height: 50px; 
	 color: white; 
    font-size: 24px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px;
    width: 50px;  
    height: 50px;  
	 color: white;  
    font-size: 24px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9; 
    border-radius: 25px; 
    width: 50px;  
    height: 50px;  
	 color: white; 
    font-size: 24px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFD700; 
    border-radius: 25px; 
    width: 50px;  
    height: 50px; 
	 color: white;  
    font-size: 27px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px;  
    width: 50px; 
    height: 50px;  
	 color: white; 
    font-size: 24px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px;  
    width: 50px; 
    height: 50px;  
	 color: white;  
    font-size: 24px; 
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9; 
    border-radius: 25px; 
    width: 50px;  
    height: 50px;  
	 color: white; 
    font-size: 24px;  
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equal_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FFD700; 
    border-radius: 25px; 
    width: 50px; 
    height: 50px; 
	 color: white;  
    font-size: 23px; 
    font-weight: bold;  
}
</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decimal_point">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px; 
    width: 50px;  
    height: 50px;  
	 color: white; 
    font-size: 24px;  
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digit_0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #a9a9a9;  
    border-radius: 25px; 
    width: 50px; 
    height: 50px;  
	 color: white; 
    font-size: 24px; 
    font-weight: bold; 
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
